changelogs/unreleased/2802-ashish-amarnath

```diff
@@ -0,0 +1,2 @@
+🐛 Supply command to run restic-wait init container
+
```

Comment:
Remove blank line; this will introduce unnecessary line breaks when we concatenate the changelogs at release time.